sklearn/ensemble/tests/test_bagging.py

```diff
@@ -663,3 +663,21 @@ def test_oob_score_removed_on_warm_start():
     clf.fit(X, y)
 
     assert_raises(AttributeError, getattr, clf, "oob_score_")
+
+
+def test_consistent_index_sampling():
```

Comment:
Do you mean make a new test where we train two separate estimators of the same type on identical data with identical random state and then see if prediction results, or oob scores are the same?